source/overview/faq.rst

```diff
@@ -31,6 +31,79 @@ We test to 60,000 concurrent users regularly, and have a peak concurrent utiliza
 
 Mattermost provides an open source, well-documented load test simulator to verify that your Mattermost deployment can achieve the stated scale benchmarks ahead of production deployment. 
 
+What features are available on different Mattermost clients?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following chart highlights the end user features of Mattermost and their support across Web, Desktop and Mobile applications (iOS and Android).
+
+.. csv-table::
+    :header: "Feature", "Web", "Desktop", "Mobile"
+
+    "**Channels**"
```

Comment:
In addition to bolding, propose also capitalizing so the section headers are more clear.